ci-operator/step-registry/ipi/conf/vsphere/ipi-conf-vsphere-commands.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+TFVARS_PATH=/var/run/secrets/ci.openshift.io/cluster-profile/secret.auto.tfvars
+vsphere_user=$(grep -oP 'vsphere_user="\K[^"]+' ${TFVARS_PATH})
+vsphere_password=$(grep -oP 'vsphere_password="\K[^"]+' ${TFVARS_PATH})
+
+declare -a vips
+mapfile -t vips < "${SHARED_DIR}/vips.txt"
+
+cat >> "${CONFIG}" << EOF
+baseDomain: origin-ci-int-aws.dev.rhcloud.com
```

Comment:
should we be harcoding this?

Reply:
> should we be harcoding this?

I think so, based on:
https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh#L23
and
https://github.com/openshift/release/blob/master/ci-operator/templates/openshift/installer/cluster-launch-installer-upi-e2e.yaml#L1137
The second example allows baseDomain to be overriden... should we do that?

Reply:
> should we be harcoding this?

That's the existing pattern, e.g. [here][1].

[1]: https://github.com/openshift/release/blob/5a497f22f616ed782a3855051d746930f40b42ee/ci-operator/step-registry/ipi/conf/aws/ipi-conf-aws-commands.sh#L23